Explain the 2 different windows download options (#4141)
ollama/ollama: docs/windows.md

- `explorer %LOCALAPPDATA%\Programs\Ollama` contains the binaries (The installer adds this to your user PATH)
- `explorer %HOMEPATH%\.ollama` contains models and configuration
- `explorer %TEMP%` contains temporary executable files in one or more `ollama*` directories

> ## Standalone binary

Reviewer: ```suggestion
## Standalone CLI
```

The easiest way to install Ollama on Windows is to use the `OllamaSetup.exe`
installer. It installs in your account without requiring Administrator rights.
We update Ollama regularly to support the latest models, and this installer will
help you keep up to date.

> If you'd like to install or integrate Ollama as a service, a separate

Reviewer: ```suggestion
If you'd like to install or integrate Ollama as a service, a standalone
```

- `explorer %LOCALAPPDATA%\Programs\Ollama` contains the binaries (The installer adds this to your user PATH)
- `explorer %HOMEPATH%\.ollama` contains models and configuration
- `explorer %TEMP%` contains temporary executable files in one or more `ollama*` directories

## Multi-user Advanced Scenarios

The `OllamaSetup.exe` installer has been designed for individual use on your
Windows PC.  It installs in your account without requiring Administrator rights.
We update Ollama regularly to support the latest models, and this installer will
help you keep up to date.

> If you have a multi-user system, or are looking for more control over where

Reviewer 2: Maybe:

If you want to run Ollama as a system service because you have a multi-user system, or are looking for more control over where Ollama is installed, we provide a separate `ollama-windows-amd64.zip` file. This file contains the Ollama CLI and GPU library dependencies for Nvidia and AMD. It does not include the tray application and will not notify you when there is a new version of Ollama.

Ollama is installed, and want to run it as a system service, we also provide an
`ollama-windows-amd64.zip` file which contains the ollama CLI and GPU library
dependencies for Nvidia and AMD.  This does not include the tray application, or

> update logic to keep your install current.  You can use a tool like

Reviewer 2: Maybe:

Use tools such as [NSSM](https://nssm.cc/) to run `ollama serve` as a system service.

- `explorer %LOCALAPPDATA%\Programs\Ollama` contains the binaries (The installer adds this to your user PATH)
- `explorer %HOMEPATH%\.ollama` contains models and configuration
- `explorer %TEMP%` contains temporary executable files in one or more `ollama*` directories

> ## Multi-user Advanced Scenarios

Reviewer: ```suggestion
## Standalone binary
```

> The `OllamaSetup.exe` installer has been designed for individual use on your

Reviewer: ```suggestion
The easiest way to install Ollama on Windows is to use the `OllamaSetup.exe` installer.
```

Windows PC.  It installs in your account without requiring Administrator rights.
We update Ollama regularly to support the latest models, and this installer will
help you keep up to date.

> If you want to run Ollama as a system service because you have a multi-user
> system, or are looking for more control over where Ollama is installed, we
> provide a separate `ollama-windows-amd64.zip` file. This file contains the Ollama
> CLI and GPU library dependencies for Nvidia and AMD. It does not include the
> tray application and will not notify you when there is a new version of Ollama.
> Use tools such as [NSSM](https://nssm.cc/) to run `ollama serve` as a system
> service.

Reviewer: ```suggestion
If you'd like to install or integrate Ollama as a service, a separate `ollama-windows-amd64.zip` zip file is available containing only the Ollama CLI and GPU library dependencies for Nvidia and AMD. This allows for embedding Ollama in existing applications, or running it as a system service via `ollama serve` with tools such as [NSSM](https://nssm.cc/).
```
